content/applications/finance/payment_providers/demo.rst

```diff
@@ -0,0 +1,129 @@
+=====================
+Demo payment provider
+=====================
+
+Odoo's **Demo Payment Provider** allows you to test business flows involving online transactions
+without requiring real banking credentials.
+
+.. seealso::
+   :doc:`../payment_providers`
+
+Installation
+------------
+
+Go to :menuselection:`Accounting or Website --> Configuration --> Payment Providers` and click
+:guilabel:`Install` on the :guilabel:`Demo` payment provider. Then, switch its state to
+:guilabel:`Test Mode`, and select a :guilabel:`Payment Journal`.
+
+.. image:: demo/demo-payment-provider.png
+   :align: center
+   :alt: Demo Payment Provider icon.
+
+Configuration
+-------------
+
+The **Demo Payment Provider** has multiple configuration options that can be accessed by going to
+:menuselection:`Accounting or Website app --> Configuration --> Payment Providers`.
```

Comment:
See this review's comment for more explanation about this.
Also, even if it works, we made all headings markups consistent in this repository, please use the same ones as described here
https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#headings
```suggestion
Configuration
=============

.. seealso::
   :ref:`payment_providers/add_new`

.. important::
   switch its state to :guilabel:`Test Mode`.
```